corehq/apps/hqwebapp/two_factor_gateways.py

```diff
@@ -59,10 +62,18 @@ def send_sms(self, device, token):
             return HttpTooManyRequests()
 
         message = _('Your authentication token is %s') % token
-        self.client.api.account.messages.create(
-            to=device.number.as_e164,
-            from_=self.from_number,
-            body=message)
+        try:
+            self.client.api.account.messages.create(
+                to=device.number.as_e164,
+                from_=self.from_number,
+                body=message)
+        except TwilioRestException as e:
+            request = get_request()
+            _soft_assert = soft_assert(notify_admins=True)
+            _soft_assert(False, msg='TwilioRestException', obj=str(e))
```

Comment:
instead of soft assert can we do `notify_exception` which will sent the error to sentry

Reply:
fixed